### DISCUSSION TITLE
I agree on this, it's confusing.

### DISCUSSION THREAD

**Discussion by cristianofmc** [twbs/bootstrap#32404] [Q&A]
I agree on this, it's confusing.

The link of example  not work
https://jsbin.com/zopod/1/edit?html,css,js

_Originally posted by @gustavohenke in https://github.com/twbs/bootstrap/issues/4215#issuecomment-8215829_

**Comment by patrickhlauke** [comment]
instead of just obliquely referencing a comment from 8 years ago, I'd suggest formulating your question (and titling your issues) a bit more understandably.

moved this to discussions, as it appears to be a how-to/why isn't this working type query
